schedule/yast/sle/flows/default_sle15sp4_x86.yaml

```diff
@@ -1,31 +1,51 @@
 ---
-name: yast-mru-install_dev
-vars:
-  DUD_ADDONS: sdk
-  YUI_REST_API: 1
-schedule:
+# Default ordered sequence of steps for sle15sp4 with maintenance updates
+bootloader:
   - installation/bootloader_start
+setup_libyui:
   - installation/setup_libyui
+product_selection:
   - installation/product_selection/install_SLES
+self_update: []
+license_agreement:
   - installation/licensing/accept_license
+registration:
   - installation/registration/register_via_scc
+extension_module_selection:
   - installation/module_registration/register_extensions_and_modules
   - installation/module_registration/add_additional_regcodes
   - installation/module_registration/import_untrusted_gnpupg_key
+additional_products:
   - installation/add_on_product_installation/add_additional_products
```

Comment:
Seems broken this in development group.
https://openqa.suse.de/tests/10046881#dependencies
Could you please check it?

Reply:
Because the qcow file doesn't exist now. This 'sle-15-SP4-x86_64-151.1-textmode@64bit.qcow2' change name to 'sle-15-SP4-x86_64-GM-textmode@64bit.qcow2'  I will update it

Reply:
Done